token/core/zkatdlog/nogh/tokens.go

```diff
@@ -54,7 +54,7 @@ func NewTokensService(publicParametersManager common.PublicParametersManager[*cr
 
 	supportedTokenFormatList := make([]token.Format, 0, 3*len(pp.IdemixIssuerPublicKeys))
 	for _, key := range pp.IdemixIssuerPublicKeys {
-		for _, precision := range crypto.SupportedPrecisions {
+		for _, precision := range v1.SupportedPrecisions {
```

Comment:
Is this code related to `v1` only?

Reply:
Maybe everything that is not under `v1` should not have references to `v1` but to interfaces or common structs?